### PR TITLE
Fix #8930: Change the font of Urdu locale to Noto Nastaliq

### LINKED ISSUE #8930: To change the font of Urdu locale to Noto Nastaliq
This is a request to change the font on [Mozilla.org](https://www-dev.allizom.org/ur/) to [Noto Nastaliq](https://fontinfo.opensuse.org/fonts/NotoNastaliqUrduRegular.html) from the current font?

Fonts:
[Noto Nastaliq](https://fontinfo.opensuse.org/fonts/NotoNastaliqUrduRegular.html)
[Noto Nastaliq](https://github.com/googlefonts/noto-fonts/tree/master/hinted/NotoNastaliqUrdu)

As the current font is best for Arabic but not for Urdu readability in the regions of Pakistan and India. As most of the Urdu Books we read are on Noto Nastaliq.

I am one of the [Mozilla L10n Urdu Team Manager](https://pontoon.mozilla.org/ur/), and this is also a request from all of the Urdu volunteers working on Pontoon.

Thanks

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

**Review by alexgibson — APPROVED** [review]
r+

**Comment by alexgibson** [comment]
Hmm, ok - the code here and everything looks fine, but I did notice last second some font errors / warnings in the console:

```
downloadable font: GSUB: Bad multiple subst table format 9886 (font-family: "X-LocaleSpecific" style:normal weight:700 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Bold.woff2
downloadable font: Layout: Failed to parse lookup subtable 1 (font-family: "X-LocaleSpecific" style:normal weight:700 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Bold.woff2
downloadable font: Layout: Failed to parse subtable 0 (font-family: "X-LocaleSpecific" style:normal weight:700 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Bold.woff2
downloadable font: Layout: Failed to parse lookup 14 (font-family: "X-LocaleSpecific" style:normal weight:700 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Bold.woff2
downloadable font: GSUB: Failed to parse lookup list table (font-family: "X-LocaleSpecific" style:normal weight:700 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Bold.woff2
downloadable font: GSUB: Failed to parse table (font-family: "X-LocaleSpecific" style:normal weight:700 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Bold.woff2
downloadable font: rejected by sanitizer (font-family: "X-LocaleSpecific" style:normal weight:700 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Bold.woff2
downloadable font: GSUB: Bad multiple subst table format 9886 (font-family: "X-LocaleSpecific" style:normal weight:400 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Regular.woff2
downloadable font: Layout: Failed to parse lookup subtable 1 (font-family: "X-LocaleSpecific" style:normal weight:400 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Regular.woff2
downloadable font: Layout: Failed to parse subtable 0 (font-family: "X-LocaleSpecific" style:normal weight:400 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Regular.woff2
downloadable font: Layout: Failed to parse lookup 14 (font-family: "X-LocaleSpecific" style:normal weight:400 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Regular.woff2
downloadable font: GSUB: Failed to parse lookup list table (font-family: "X-LocaleSpecific" style:normal weight:400 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Regular.woff2
downloadable font: GSUB: Failed to parse table (font-family: "X-LocaleSpecific" style:normal weight:400 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Regular.woff2
downloadable font: rejected by sanitizer (font-family: "X-LocaleSpecific" style:normal weight:400 stretch:100 src index:0) source: http://localhost:8000/media/fonts/NotoNastaliqUrdu-Regular.woff2 
```

These might be fine, but I'm not 100% sure.

Any ideas here @craigcook? ^

**Comment by craigcook** [comment]
That seems to be a problem in the font files, maybe corrupted when converting from TTF to WOFF. I downloaded them from Google Fonts and converted them with https://cloudconvert.com/ttf-converter and now I don't get those errors locally with the new files.

File: media/css/l10n/ur/intl.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* Issue 8930 */
+
+@font-face {
+    font-family: X-LocaleSpecific;
+    font-weight: normal;
+    font-display: swap;
+    src:
+        url('/media/fonts/NotoNastaliqUrdu-Regular.woff2') format('woff2'),
```
Comment (craigcook) [inline media/css/l10n/ur/intl.css:8]: I'd suggest putting these files in `/media/fonts/l10n/ur/`

**Comment by robhudson** [comment]
Updated

I used the cloudconvert site to create the `.woff` and `.woff2` files and placed them under the l10n font folder.

**Review by craigcook — APPROVED** [review]
🦦

**Pull request merged**